### PR TITLE
Global css standarization

### PR DESCRIPTION
**Pull request by shivani170**
# Description

To add prefix `dc__` to All the global css in base.css component

Fixes https://github.com/devtron-labs/devtron/issues/2036

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/ciPipeline/ciPipeline.css [hunk 1]
```diff
@@ -160,7 +160,7 @@
     margin-right: 10px;
 }
 
-.info__container.info__container--linked-ci {
+.dc__info__container.info__container--linked-ci {
```
Comment (vivek-devtron) [inline src/components/ciPipeline/ciPipeline.css:163]: can we wrap it in some parent container class?

File: src/components/app/create/createApp.css [hunk 2]
```diff
@@ -1,65 +0,0 @@
-.create-app-modal {
```
Comment (vivek-devtron) [inline src/components/app/create/createApp.css:1]: Is removal of this intentional?

File: src/css/base.scss [hunk 3]
```diff
@@ -46,42 +47,42 @@ html {
     border-radius: 4px;
 }
 
-.info__container.info__container--documentation-link {
+.dc__info__container.info__container--documentation-link {
```
Comment (vivek-devtron) [inline src/css/base.scss:50]: we should follow the same convention for child CSS

File: src/css/base.scss [hunk 4]
```diff
@@ -46,42 +47,42 @@ html {
...
 
-.chart-grid-item__icon {
+.dc__chart-grid-item__icon {
     object-fit: contain;
     max-width: 100%;
     max-height: 100%;
 }
 
-.chart-grid-item__icon.list-icon {
+.dc__chart-grid-item__icon.list-icon {
```
Comment (vivek-devtron) [inline src/css/base.scss:85]: Same here

File: src/css/base.scss [hunk 5]
```diff
@@ -2060,3 +1949,127 @@ textarea {
...
+  margin-left: 0px !important;
+  margin-right: 0px !important;
+}
+
+.p-lr-20 {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.m-b-4 {
```
Comment (sohel-rp) [inline src/css/base.scss:2069]: use `mb-4-imp` instead

File: src/components/EnvironmentOverride/ConfigMapOverrides.tsx [hunk 6]
```diff
@@ -447,11 +447,11 @@ const OverrideConfigMapForm: React.FC<ConfigMapProps> = memo(function OverrideCo
                         </div>
                     </div>
                     {!name && external ? (
-                        <div className="info__container mb-24">
+                        <div className="dc__info__container mb-24">
```
Comment (sohel-rp) [inline src/components/EnvironmentOverride/ConfigMapOverrides.tsx:450]: Change it to `dc__info-container` for consistency

File: src/components/EnvironmentOverride/ConfigMapOverrides.tsx [hunk 7]
```diff
@@ -447,11 +447,11 @@ const OverrideConfigMapForm: React.FC<ConfigMapProps> = memo(function OverrideCo
...
                             <Info />
                             <div className="flex column left">
-                                <div className="info__title">Using External Configmaps</div>
-                                <div className="info__subtitle">
+                                <div className="dc__info__title">Using External Configmaps</div>
+                                <div className="dc__info__subtitle">
```
Comment (sohel-rp) [inline src/components/EnvironmentOverride/ConfigMapOverrides.tsx:454]: same here

File: src/components/EnvironmentOverride/SecretOverrides.tsx [hunk 8]
```diff
@@ -568,11 +568,11 @@ export function OverrideSecretForm({ name, appChartRef, toggleCollapse }) {
...
                     </div>
                     {externalType === 'KubernetesSecret' ? (
-                        <div className="info__container mb-24">
+                        <div className="dc__info__container mb-24">
                             <Info />
                             <div className="flex column left">
-                                <div className="info__title">Using External Secrets</div>
-                                <div className="info__subtitle">
+                                <div className="dc__info__title">Using External Secrets</div>
+                                <div className="dc__info__subtitle">
```
Comment (sohel-rp) [inline src/components/EnvironmentOverride/SecretOverrides.tsx:575]: same here

File: src/components/app/create/CreateApp.tsx [hunk 9]
```diff
@@ -420,11 +420,11 @@ export class AddNewApp extends Component<AddNewAppProps, AddNewAppState> {
                             </div>
                         )}
                         {this.state.form.appCreationType === AppCreationType.Existing && (
-                            <div className="info__container info__container--create-app eb-2 mb-16">
+                            <div className="dc__info__container info__container--create-app eb-2 mb-16">
                                 <Info />
                                 <div className="flex column left">
                                     <div>
-                                        <div className="info__title">Important: </div>Do not forget to modify git
+                                        <div className="dc__info__title">Important: </div>Do not forget to modify git
```
Comment (sohel-rp) [inline src/components/app/create/CreateApp.tsx:427]: same here

File: src/components/ciPipeline/ciPipeline.css (hunk 1 above)
Comment (shivani170) [reply src/components/ciPipeline/ciPipeline.css:163]: removed this and added mb-16 manually

File: src/components/app/create/createApp.css (hunk 2 above)
Comment (shivani170) [reply src/components/app/create/createApp.css:1]: yes converted to scss

**Pull request merged**